frontend/src/AlertConfiguration/AlertConfigurationComponents/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, Fragment } from 'react';
```

Comment:
again, this file seems to look very similar to `ConfigForm.js` and `ConfigCard.js` and `ConfigList.js` and `ControlPanel.js`...

would love to know the reason behind the similarities :) thanks!

Reply:
Thank you for pointing this issue out. I must of made some mistake when I was copying and pasting between the react app I use to edit and the step folder. All of my files were actually duplicates. I have just fixed this issue.